tests/unit/serializer.test.ts

```diff
@@ -0,0 +1,17 @@
+import Elder, { Serializer, Model, type } from '../../src'
+const config = { adapters: { default: { database: 'ash' } } }
+
+describe('Serializers', () => {
+  test('Serializer base class serialize() static method', () => {
+    const serializer = Serializer.create()
+    class Cat extends Model {
+      @type('string') fizz: string
+    }
+    const orm = Elder.create({ config, models: { cat: Cat } })
```

Comment:
Unless there's a perf reason for defining all models up front, you might consider changing the design of `Elder` to either have an `Elder.prototype.addModel` method or some other mechanism in `Model` that will latch onto the Elder instance somehow. This test pain is definitely exposing pain that would be leaking to your users, perhaps unnecessarily